### DISCUSSION TITLE
Missing Spans for Lettuce Instrumentation

### DISCUSSION THREAD

**Discussion by Taragos** [open-telemetry/opentelemetry-java-instrumentation#10150] [Q&A]
Hello,

When using the current version of the opentelemetry-javaagent (1.32.0) to automatically instrument the lettuce library, I receive the following warning (as well as a few more that concern future instrumentation) when the debug mode is activated.

`Skipping future instrumentation for io.lettuce.core.protocol.AsyncCommand`

When checking my traces in Jaeger, I also don't see any traces concerning Redis calls (logs show that the Redis is being accessed for setting or retrieving data). 

Is this a known error and if yes is there a workaround?

**Comment by laurit** [comment]
> Skipping future instrumentation for io.lettuce.core.protocol.AsyncCommand

This is not a warning but a debug statement, it does not indicate that anything is wrong.

> When checking my traces in Jaeger, I also don't see any traces concerning Redis calls (logs show that the Redis is being accessed for setting or retrieving data).

Please provide a minimal sample application that reproduces your issue.
